features/eolearn/tests/test_clustering.py

```diff
@@ -1,3 +1,12 @@
+"""
+Credits:
+Copyright (c) 2020 Beno Šircelj (Josef Stefan Institute)
```

Comment:
here as well

Reply:
just check everywhere for this kind of docstring, seems to be this one everywhere